diff --git a/node_modules/ngx-markdown/fesm2020/ngx-markdown.mjs b/node_modules/ngx-markdown/fesm2020/ngx-markdown.mjs
index e1922f2..47ab9fb 100644
--- a/node_modules/ngx-markdown/fesm2020/ngx-markdown.mjs
+++ b/node_modules/ngx-markdown/fesm2020/ngx-markdown.mjs
@@ -103,6 +103,7 @@ class MarkdownService {
         }
     }
     renderKatex(html, options) {
+        console.log("HEY!");
         if (!isPlatformBrowser(this.platform)) {
             return html;
         }
